paasta_tools/cli/cmds/status.py

```diff
@@ -919,6 +919,15 @@ def print_kubernetes_status(
         output.append("      Pods:")
         pods_table = format_kubernetes_pod_table(kubernetes_status.pods)
         output.extend([f"        {line}" for line in pods_table])
+
+    if kubernetes_status.smartstack is not None:
+        smartstack_status_human = get_smartstack_status_human(
+            kubernetes_status.smartstack.registration,
+            kubernetes_status.smartstack.expected_backends_per_location,
+            kubernetes_status.smartstack.locations,
+            getattr(kubernetes_status.smartstack, "error_message", None),
```

Comment:
Pretty sure you don't need this `getattr` -- I just had that in there originally because I added the error message later and didn't want to cause errors during deployment if the server didn't return this field.

Reply:
Cool, I'll change that.